hoomd/pytest/test_table.py

```diff
@@ -154,7 +166,13 @@ def test_only_string_and_scalar_quantities(device):
     logger = hoomd.logging.Logger()
     output = StringIO("")
     with pytest.raises(ValueError):
-        _ = hoomd.write.Table(0, logger, output)
+        _ = hoomd.write.Table(1, logger, output)
     logger = hoomd.logging.Logger(categories=['sequence'])
     with pytest.raises(ValueError):
-        _ = hoomd.write.Table(0, logger, output)
+        _ = hoomd.write.Table(1, logger, output)
```

Comment:
Some linters will complain about this syntax and prefer that the output value is never assigned (because it's not used). Assignment to a variable cannot have side-effects as far as I am aware, so the test behavior of raising an error should be equivalent.
```suggestion
        hoomd.write.Table(1, logger, output)
```